### DISCUSSION TITLE
Show arguments on stack trace

### DISCUSSION THREAD

**Discussion by agustinprod** [facade/ignition#178] [Ideas] [answered]
With whoops, you can see all the args by stack trace level. It helps a lot and save time. Can we have this on ignition?

![example](https://i1.wp.com/wp.laravel-news.com/wp-content/uploads/2017/06/laravel-55-whoops.png?resize=2200%2C1125)

**Comment by robbielove** [comment]
This might be helpful: https://github.com/filp/whoops/pull/387

**Comment by jasonvarga** [comment]
Is this on the roadmap? It's the one thing I'm missing from Whoops.
If it's not planned for any time soon, I'd be happy to attempt a PR.

**Comment by freekmurze** [comment]
@jasonvarga I'd be grateful for a PR for this! 👍

**Comment by MrFiregore** [comment]
any news?

**Reply by jasonvarga** [reply]
It was more complicated than I originally thought and I haven't had a chance to go back and try any more.

**Comment by agustinprod** [comment, marked as answer]
https://twitter.com/freekmurze/status/1675804846459322373

**Reply by rubenvanassche** [reply]
We've implemented this on spatie/laravel-ignition, this is an older version of ignition which isn't being maintained anymore.
